.github/ISSUE_TEMPLATE/--feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "✨Feature request"
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem?
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+## Describe the solution you'd like
+A clear and concise description of what you want to happen.
+
+## Describe alternatives you've considered
```

Comment:
why would you need to describe alternatives?

Reply:
This would be something like
> Currently, I have to do step 1 and step 2 to achieve my goal, but with this feature, it would be only one step